### ISSUE TITLE
having some trouble with Layout/SpaceAroundOperators

### ISSUE THREAD

**Issue by InteNs** [bbatsov/rubocop#4579]
We're implementing rubocop in our projects at work and I've stumbled upon an issue and i'm not sure what's best to do here.

--------

## Expected behavior

this should not offend by rubocop:
```ruby
  columns [:wagon, :wagon_nr]                       =>
            [[
              'wagon',
              'wagonnummer',
              'wagon nummer',
              'wagon nr',
              'wagon number',
              'wagonnumber',
              'wagennummer',
              'waggonnummer',
              'wgnr12stellig',
              'Železniční vůz / waggon'
            ]],
          [:wagon, :wagon_sub_type_id]              =>
            [[
              'gattung',
              'wagontype',
              'gat uic',
              'type'
            ]],
          [:wagon_achievements, :loaded_distance]   => ['loaded km'],
          [:wagon_achievements, :unloaded_distance] => ['unloaded km'],
          [:wagon_achievements, :cleaned]           => ['cleaned']
```

## Actual behavior
```sh
app/parsers/assets/wagons/wcl_parser.rb:2:53: C: Operator => should be surrounded by a single space.
  columns [:wagon, :wagon_nr]                       =>
                                                    ^^
```

so rubocop only marks the first operator as offending, the other operators are fine

## Steps to reproduce the problem

put the source above in a file and:
`rubocop /path/to/file`

## RuboCop version
```
-> rubocop -V
0.49.1 (using Parser 2.4.0.0, running on ruby 2.3.1 x86_64-darwin16)
```

**Comment by jonas054** [comment]
The `Layout/SpaceAroundOperators` cop works in co-operation with the `Layout/AlignHash` cop, and uses its configuration. To allow the style you're using here, you need to set
```yaml
Layout/AlignHash:
  EnforcedHashRocketStyle: table
```

**Comment by mikegee** [comment]
Does @jonas054's comment resolve this issue?
